api/assets/pdf/visit_report.php

```diff
@@ -27,7 +27,7 @@
         <p>Registered Users</p>
         <ul>
             <?php foreach($users as $u): ?>
-                <li><?php echo $u['FULLNAME'] ?></li>
+                <li><?php echo $u['TITLE'] ?> <?php echo $u['GIVENNAME'] ?> <?php echo $u['FAMILYNAME'] ?></li>
```

Comment:
This comes from SQL above it and will never include TITLE so don't include it here. Does it include the next property too (LC) I think that might crash it as well

Reply:
I believe the SQL is at https://github.com/DiamondLightSource/SynchWeb/blob/9a6f065575411e0036c0800d9ade20520f156b55/api/src/Page/PDF.php#L217 which includes TITLE. LC is set on line 205 of the same file as part of $info.